test/CMakeLists.txt

```diff
@@ -1,5 +1,16 @@
 if (BUILD_TESTS)
 
+    cmake_policy(SET CMP0064 NEW)
+    cmake_policy(SET CMP0057 NEW)
```

Comment:
https://cmake.org/cmake/help/v3.3/policy/CMP0057.html
The NEW behavior is to interpret the IN_LIST operator.
This policy was introduced in CMake version 3.3.